js/letters.js

```diff
@@ -17,9 +17,21 @@ var counter = {
 
 function countLetters(counter, sample_text){
   // FIX ME
+
+  let text = sample_text.toLowerCase();
+  if(sample_text.length === 0){
+    return counter;
+
+  }if([text.slice(0,1)] ){
```

Comment:
The square brackets here are not necessary and actually makes the if statement true at all times. 

You'll also need to figure out if the particular character you're getting is in the counter object or not. Check out the object.hasOwnProperty method for a way to double check if a key exists in an object.